package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "opencodelists",
+  "scripts": {
+    "build": "webpack --mode production",
+    "watch": "webpack --watch --mode development"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.4",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "babel-loader": "^8.1.0",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
+  },
+  "dependencies": {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ebmdatalab/opencodelists.git"
+  },
+  "license": "MIT"
```

Comment:
I'm pretty sure this is just an npm default, but I don't know if we've decided on the license yet so should probably leave this out.